package.json

```diff
@@ -66,9 +66,14 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "testem ci",
-    "lint-dts": "tslint sweetalert2.d.ts --format verbose",
-    "check-dts": "tsc sweetalert2.d.ts && npm run lint-dts",
+    "start": "gulp watch",
+    "fix:lint": "standard --fix",
+    "test": "npm run build && npm run check",
+    "build": "gulp build",
```

Comment:
`gulp build` rather than just `gulp` because it's nice to be explicit

Reply:
:+1: